x-pack/plugins/security/public/management/roles/model/kibana_privilege.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+
+export class KibanaPrivilege {
+  constructor(public readonly id: string, public readonly actions: string[] = []) {}
+
+  public get name() {
+    return _.capitalize(this.id);
+  }
+
+  public grantsPrivilege(candidatePrivilege: KibanaPrivilege) {
```

Comment:
I can't find anywhere that we're using action wildcards at the moment, but there are references to them in `x-pack/plugins/security/server/authorization/actions` that we should likely remove, and we'll have to ensure we don't use them in the future.

Reply:
Good catch, I'll get this cleaned up and tested appropriately